asv_bench/benchmarks/pandas/pandas_vb_common.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Where do we plan to use the content of this file?

Reply:
Let's add it as is for now. We don't know the final set of benchmarks yet. After adding - will edit.

Reply:
Okay